rust/operator-binary/src/superset_controller.rs

```diff
@@ -91,6 +107,18 @@ pub enum Error {
     ObjectMissingMetadataForOwnerRef {
         source: stackable_operator::error::Error,
     },
+    #[snafu(display("superset only supports a single authentication method"))]
```

Comment:
```suggestion
    #[snafu(display("Superset only supports a single authentication method"))]
```

We still capitalize proper names in error messages elsewhere